### DISCUSSION TITLE
combine SSR with CSR

### DISCUSSION THREAD

**Discussion by Amir3599** [jaredpalmer/razzle#1844] [Q&A]
Hi ,
I want to create a website which CEO is important for the the first page and other pages is like dashboard and no need CEO,
can I **combine** Server-Side-Rendering and Client-Side-Rendering with **Razzle** on this case?

**Comment by fivethreeo** [comment]
You mean SEO? If so, yes.

**Reply by udit1994** [reply]
@fivethreeo Could you help me understand how would we do that?

**Comment by jleei** [comment]
@udit1994 The function of fetching data is not called on the server side to achieve client side rendering. Take a look at this site, [gainingcoin.com](https://gainingcoin.com/) (built with Razzle), their user Profile page is CSR, all other pages are SSR.

**Comment by openmind-why** [comment]
中间件拦截csr页面路由
